tests/unittests/BackendTestUtils2.cpp

```diff
@@ -81,13 +81,14 @@ profileAndGetNodeQuantizationInfo(CreateAndInitFunction createAndInitFunction,
   LoweredInfoMap loweredMapForProf;
   PlaceholderBindings pBindings;
   ExecutionEngine2 PEE{"Interpreter"};
-  FunctionTensorPair PFT = createAndInitFunction(pBindings, PEE);
+  createAndInitFunction(pBindings, PEE);
   CompilationContext cctx{&pBindings, &loweredMapForProf};
   cctx.precisionConfig.quantMode = QuantizationMode::Profile;
   PEE.compile(cctx);
   PEE.run(pBindings);
-  return quantization::generateNodeQuantizationInfos(pBindings, PFT.first,
-                                                     loweredMapForProf, schema);
+  return quantization::generateNodeQuantizationInfos(
+      pBindings, PEE.getModule().getFunctions().front(), loweredMapForProf,
```

Comment:
Any particular reason you changed this code?

Reply:
After compilation in quantization mode the original Function no longer exisits, the Partitioner creates a new function in it's quantization flow and deletes the old one. I'll add a comment to clarify.